tests/Portal.test.tsx

```diff
@@ -1,43 +1,44 @@
 import React from 'react';
-import { mount } from 'enzyme';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
```

Comment:
This can be moved to jest.config

https://www.npmjs.com/package/@testing-library/jest-dom#user-content-usage